packages/cli/src/commands/hydrogen/deploy.ts

```diff
@@ -77,13 +78,14 @@ export default class Deploy extends Command {
     }),
     'no-json-output': Flags.boolean({
       description:
-        'Prevents the command from creating a JSON file containing the deployment URL (in CI environments).',
+        'Prevents the command from creating a JSON file containing the deployment URL in CI environments.',
```

Comment:
Actually, I never updated this, but the JSON file potentially contains more than just the URL.
```suggestion
        'Prevents the command from creating a JSON file containing the deployment details in CI environments.',
```